src/aiida/orm/comments.py

```diff
@@ -65,44 +67,29 @@ class Comment(entities.Entity['BackendComment', CommentCollection]):
 
     _CLS_COLLECTION = CommentCollection
 
-    __qb_fields__ = [
-        add_field(
-            'uuid',
-            dtype=str,
-            is_attribute=False,
-            doc='The UUID of the comment',
-        ),
-        add_field(
-            'ctime',
-            dtype=datetime,
-            is_attribute=False,
-            doc='Creation time of the comment',
-        ),
-        add_field(
-            'mtime',
-            dtype=datetime,
-            is_attribute=False,
-            doc='Modified time of the comment',
-        ),
-        add_field(
-            'content',
-            dtype=str,
+    class Model(entities.Entity.Model):
+        uuid: Optional[str] = MetadataField(
+            description='The UUID of the comment', is_attribute=False, exclude_to_orm=True
+        )
+        ctime: Optional[datetime] = MetadataField(
+            description='Creation time of the comment', is_attribute=False, exclude_to_orm=True
+        )
+        mtime: Optional[datetime] = MetadataField(
+            description='Modified time of the comment', is_attribute=False, exclude_to_orm=True
+        )
+        node: int = MetadataField(
+            description='Node PK that the comment is attached to',
```

Comment:
`node` as an `int` type might be confusing, no? One might expect a `Node` type. Was `node_pk`, so I guess you had a reason to go with just `node`?